unicorecmsfflangola/static/css/style.css

```diff
@@ -17,6 +17,10 @@
     #header { padding: 10px; background: #f9f2e2; text-align: center; border-bottom: 2px solid #e3d2b8; }
         #header img { display: block; margin: 0 auto; width: 140px; }
 
+/*  LANG  */
+    .lang { padding: 5px 16px; text-align: right; color: #fff; background: #009ee3;}
```

Comment:
@nathanbegbie: looking good, but I think `.lang color` should be `#6cc9fd` like the about, etc menu.